workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -43,7 +56,9 @@ class UserConfig:
     scikit_learn: Optional[MachineLearningConfig] = None
     nudging: Optional[NudgingConfig] = None
     tendency_prescriber: Optional[TendencyPrescriberConfig] = None
-    online_emulator: Optional[runtime.emulator.Config] = None
+    physics_machine_learning: Optional[
```

Comment:
Approving since we did discuss this yesterday, but the configuration still feels a little off to me and prone to changing in the future as we add more ways of interacting with the physics:

1. still unclear to me why predicting a tendency  (fv3fit) vs a state update (emulator) cannot be done simultaneously, is there some conceptual reason for why this should be impossible? Ultimately, I expect the fv3fit and emulator models will share a common serialized format. Another example is that we may want to use the emulator's masking code for the "fv3fit" models. Can we structure the config in such a way that these future changes could be backward compatible? 
2. should be grouped with `tendency_prescriber` since they essentially act in similar ways. 

What do you think about the following:

```
physics:
  tendencies:
    prescribed: Optional[TendencyPrescriberConfig]
    predicted: Optional[runtime.transformers.fv3fit.Config]
  state_updates:
    predicted: Optional[runtime.emulator.Config]
```